app/ide-desktop/lib/dashboard/src/components/AriaComponents/Button/Button.tsx

```diff
@@ -87,7 +87,8 @@ export const BUTTON_STYLES = twv.tv({
       custom: 'focus-visible:outline-offset-2',
       link: 'inline-flex px-0 py-0 rounded-sm text-primary/50 underline hover:text-primary focus-visible:outline-offset-0',
       primary: 'bg-primary text-white hover:bg-primary/70 focus-visible:outline-offset-2',
-      tertiary: 'bg-share text-white hover:bg-share/90 focus-visible:outline-offset-2',
+      tertiary:
+        'relative px-new-project-button-x text-white before:absolute before:inset before:rounded-full before:bg-accent before:transition-all hover:before:brightness-90 *:relative',
```

Comment:
wtf?

Reply:
> px-new-project-button-x

It's the concern of size, not the variant

Reply:
well, we probably shouldn't have separate sizes in the first place

Reply:
Yes we should